testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -2103,6 +2148,35 @@ fn pox_integration_test() {
         eprintln!("Sort height: {}", sort_height);
     }
 
+    let pox_info = get_pox_info(&http_origin);
+
+    assert_eq!(
+        &pox_info.contract_id,
+        &format!("ST000000000000000000002AMW42H.pox")
+    );
+    assert_eq!(pox_info.first_burnchain_block_height, 0);
+    assert_eq!(pox_info.next_cycle_cur_threshold, 125080000000000);
+    assert_eq!(pox_info.cur_cycle_threshold, 125080000000000);
+    assert_eq!(pox_info.cur_cycle_stacked_ustx, 1000000000000000);
+    assert_eq!(pox_info.next_cycle_stacked_ustx, 1000000000000000);
+    assert_eq!(pox_info.reward_slots as u32, pox_constants.reward_slots());
+    assert_eq!(pox_info.next_rewards_start, 225);
+    assert_eq!(pox_info.next_prepare_phase_start, 220);
+    assert_eq!(
+        pox_info.prepare_cycle_length as u32,
+        pox_constants.prepare_length
+    );
+    assert_eq!(
+        pox_info.rejection_fraction,
+        pox_constants.pox_rejection_fraction
+    );
+    assert_eq!(pox_info.reward_cycle_id, 14);
+    assert_eq!(
+        pox_info.reward_cycle_length as u32,
+        pox_constants.reward_cycle_length
+    );
+    assert_eq!(pox_info.next_reward_cycle_in, 14);
```

Comment:
Should we be testing here whether or not the act of Stacking will bump the minimum required uSTX to lock per reward slot?

Reply:
I'm not sure -- that's tested in the coordinator integration tests (and the other PoX unit tests), and I'm a little hesitant to add much more complexity to these integration tests, because they are much more painful to iterate on.

Reply:
Fair point.  I'm fine with not testing it here as long as it's tested adequately elsewhere.